visidata/loaders/postgres.py

```diff
@@ -92,7 +92,7 @@ def reload(self):
             r = cur.fetchone()
             if r:
                 self.addRow(r)
-            self.columns = cursorToColumns(cur)
+            self.columns = cursorToColumns(cur, self)
```

Comment:
This is where we should iterate through the cursorToColumns result and call `addColumn` for each one.